frontend/src/metabase/dashboard/actions/parameters.js

```diff
@@ -109,44 +117,123 @@ export const removeParameter = createThunkAction(
   },
 );
 
-export const SET_PARAMETER_MAPPING = "metabase/dashboard/SET_PARAMETER_MAPPING";
-export const setParameterMapping = createThunkAction(
-  SET_PARAMETER_MAPPING,
-  (parameter_id, dashcard_id, card_id, target) => (dispatch, getState) => {
-    const dashcard = getState().dashboard.dashcards[dashcard_id];
-    const isVirtual = isVirtualDashCard(dashcard);
-    const isAction = isActionDashCard(dashcard);
+export function getParameterMappings(dashcard, parameter_id, card_id, target) {
```

Comment:
This is not an action so I guess it should go to ./utils right? (bonus point: that file is typescript)

What do you think about converting it to named parameters in the process? I find functions with more than 3 arguments to be confusing on the call site.
(vscode should have a refactor helper to do it)

Reply:
Yep, I moved this out to `frontend/src/metabase/parameters/utils/mapping-options.js` - it's where `getParameterMappingOptions` is, so I think they belong together